### PR TITLE
Regen meshs' stacks are now visually visible + require opening before use.

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Title

### CODE REVIEW

**Review by NoooneyDude — CHANGES_REQUESTED** [review]
The code from the other PR (#8380) to require opening an edible (like soda can) is very similar to the code here. Therefore, that and this code here should be in a new component, say: `HandPreparable` or something.

In this PR, you would not create a new component that inherits from `HealsTheLiving` but rather add a `TryGetComponent()` within `HealsTheLiving` that checks for the `HandPreparable` component says it is ready for use.

In the edible PR: similar story, checking if that component says it is ready to use.

Also, move the check to `WillInteract` if you want the interaction to propagate properly to the `HandPreparable` component. That is, the check to see if something is ready to eat / apply should be done in WillInteract (can't remember if required client-side for propagation - would need to test) and return false if not, so that the HandActivate event will propagate correctly to the `HandPreparable` component regardless of component order.

~~In light of this I'm unapproving #8380 and marking as changes required.~~ Too slow 😛

**Comment by MaxIsJoe** [comment]
Done, review pls

File: UnityProject/Assets/Scripts/Items/HandPreparable.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		{
+			var rightClickResult = new RightClickableResult();
+			if (isPrepared) return rightClickResult;
+			rightClickResult.AddElement("Open This", Open);
+			return rightClickResult;
+		}
+
+		public bool WillInteract(HandActivate interaction, NetworkSide side)
+		{
+			return isPrepared ? false : true;
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/HandPreparable.cs:46]: is this networked?

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/HandPreparable.cs:46]: no it's not, I probably should.

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/HandPreparable.cs:46]: Done

**Review by Bod9001 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**